### DISCUSSION TITLE
Bolt login password field has dissapeared in the login screen, what to do?

### DISCUSSION THREAD

**Discussion by Sander-Hub** [bolt/core#3521] [General]
# Bolt login password field has dissapeared in the login screen, what to do?

After running composer install or update it sometimes occurs that certain assets used by Bolt are missing or borked, also know as modern webdevelopment :wink:

![missing_pw-field](https://github.com/bolt/core/assets/66225295/d8a54ee2-d0ad-49aa-b905-f73717b8c075)

Run the following command on the commandline:

```
bin/console bolt:copy-assets
```

After which the password field should be back again in the Bolt login screen.

**Comment by levincem** [comment]
i've had this issue.. Empty the cache of your browser ?
